src/Annotator.scss

```diff
@@ -62,6 +62,10 @@
             vertical-align: middle;
         }
     }
+
+    @media #{$mobile} {
+        margin: 0;
```

Comment:
It's usually good practice to only override what you _need_ to: in this case, `margin-left` and `margin-right`.